rbac/rbac/settings.py

```diff
@@ -346,6 +346,15 @@
 
 ROLE_CREATE_ALLOW_LIST = ENVIRONMENT.get_value("ROLE_CREATE_ALLOW_LIST", default="").split(",")
 
+DEFAULT_DUAL_WRITE_APP_DENY_LIST = [
```

Comment:
A naming thing (which I'm fine either way with), but we do have an `exclude_apps` param in the migrator: https://github.com/RedHatInsights/insights-rbac/blob/d8051fd8ff1041f054bacdb657ce135e6b8977cc/rbac/migration_tool/migrate.py#L157C18-L157C30 I wonder does it make sense to name these `DUAL_WRITE_APP_EXCLUDE_LIST` or make them consistent?

Reply:
Love it. Thank you.

Reply:
Ended up doing a few things (will push momentarily):

- Refactored to put defaults in clowdapp as suggested
- Introduced a new environment variable to separate out app exclusion vs just resource definition exclusion (as discussed this morning)
- Renamed variables as I realized they are not exclusive to "dual write" but to v2 migration in general

Question: For defaults in the clowdapp, does this make its way into app-interface or do we need to also update configs there?

Reply:
>Question: For defaults in the clowdapp, does this make its way into app-interface or do we need to also update configs there?

We don't need to update app-interface configs. Presence in app-interface configs allows us to use same image - it overrides defaults in clowdapp yaml.